invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/detail.html

```diff
@@ -21,7 +21,7 @@
   {%- endblock css %}
 {% endif %}
 
-{%- from "invenio_app_rdm/records/macros/files.html" import file_list_box, preview_file_box, media_file_list_box %}
+{%- from "invenio_app_rdm/records/macros/files.html" import file_list_box, preview_file_box, media_file_list_box with context %}
```

Comment:
This change is necessary to pass `transfer_types` context variable down to macros referencing it. It has an impact to Jinja macros caching mechanism(disables caching). Alternative approach would be to pass `transfer_types` to every macro usage as prop...